### DISCUSSION TITLE
Visualization of deformed microstructure in 2D

### DISCUSSION THREAD

**Discussion by sajutabraham** [neperfepx/neper#520] [Q&A] [answered]
I have completed the example _**1_uniaxial**_ successfully. But how can I visualize 2D views (x-view, y-view and z-view) of deformed 3D microstructure?

**Comment by rquey** [comment, marked as answer]
Use [-slicemesh](https://neper.info/doc/neper_v.html?highlight=slicemesh#cmdoption-slicemesh). There is a (complicated) example in the [Rodrigues Color Key tutorial](https://neper.info/tutorials/orientation_color_key.html#rodrigues-color-key).

**Reply by sajutabraham** [reply]
Thank you very much.
